khal/khalendar/event.py

```diff
@@ -299,6 +299,14 @@ def symbol_strings(self) -> Dict[str, str]:
                 'right_arrow': '->'
             }
 
+    @property
+    def allowed_busystatus(self) -> Dict[str, str]:
+        return {'BUSY': 'Busy',
+                'FREE': 'Free',
+                'TENTATIVE': 'Tentative',
+                'WORKINGELSEWHERE': 'Work elsewhere',
+                'OOF': 'Out of office'}
```

Comment:
We don't need to define this dictionary inside a function, just make it a global dictionary. Otherwise it gets recreated each time the function is called, which is kinda pointless.